lms/static/js/ajax-error.js

```diff
@@ -1,5 +1,8 @@
 $(document).ajaxError(function(event, jXHR) {
     if (jXHR.status === 403) {
+        if ($('.sneakpeek-warning').length) {
```

Comment:
Basically, if we have any warning banners on t he page for nonregistered access, short-circuit and don't prompt the user to logout on (on ajax error).